tests/licensedcode/test_match_spdx_lid.py

```diff
@@ -400,7 +400,7 @@ def test_get_expression_without_and_should_not_return_unknown(self):
         unknown_symbol = get_unknown_spdx_symbol()
         line_text = '* SPDX-License-Identifier:     GPL-2.0+ BSD-2-Clause'
         expression = get_expression(line_text, licensing, spdx_symbols, unknown_symbol)
-        assert unknown_symbol != expression
+        assert expression == unknown_symbol
```

Comment:
@adityaviki Maybe you have used `==` instead of `!=`

Reply:
@rpotter12 great catch! :bow: